src/ralph/data_importer/management/commands/importer.py

```diff
@@ -71,6 +71,13 @@ def add_arguments(self, parser):
             default='utf-8',
             help="Output encoding",
         )
+        parser.add_argument(
+            '--use-imported-id',
```

Comment:
and i know that i was suggesting `use-imported-id` before, but what do you think about `map-imported-id` (or even `map-imported-id-to-new-id`) - wouldn't it be cleaner and more self-descriptive?